packages/timeline-state-resolver/src/devices/telemetrics.ts

```diff
@@ -0,0 +1,168 @@
+import {
+	DeviceOptionsTelemetrics,
+	DeviceType,
+	Mappings,
+	StatusCode,
+	TelemetricsOptions,
+	TimelineObjTelemetrics,
+} from 'timeline-state-resolver-types'
+import { TimelineState } from 'superfly-timeline'
+import { Device, DeviceStatus } from './device'
+import { Socket } from 'net'
+import * as _ from 'underscore'
+import { DoOnTime } from '../doOnTime'
+import Timer = NodeJS.Timer
+
+const TELEMETRICS_NAME = 'Telemetrics'
+const SOCKET_PORT = 5000
+const TIMEOUT_IN_MS = 2000
+
+/**
+ * Connects to a Telemetrics Device on port 5000 using a TCP socket.
+ * This class uses a fire and forget approach.
+ */
+export class TelemetricsDevice extends Device<DeviceOptionsTelemetrics> {
```

Comment:
This device does not implement `terminate()` where it should close the socket and dispose of the doOnTime.